RecoTauTag/RecoTau/BuildFile.xml

```diff
@@ -24,6 +24,8 @@
 <use   name="TrackingTools/TrackAssociator"/>
 <use   name="PhysicsTools/JetMCUtils"/>
 <use   name="CommonTools/Utils"/>
+<use   name="FastSimulation/BaseParticlePropagator"/>
```

Comment:
this is not the most desireable dependence for reco to depend on FastSimulation.
The propagation should probably be moved to CommonTools.

I have created https://github.com/cms-sw/cmssw/issues/17774